### ISSUE TITLE
duplicate cells

### ISSUE THREAD

**Issue by JoeyBodnar** [hirohisa/ImageLoaderSwift#68]
I have a list of 13 URLs that I need to load from images. I use ImageLoaderSwift to load the images into my app, but I am getting duplicates. My cellForItemAtIndexPath just reads:

cell.image.load(self.myURLs[indexPath.item])

the image below shows a few of the duplicate cells i'm talking about (there are more but off screen)

<img width="320" alt="capture d ecran 2016-01-07 a 17 15 03" src="https://cloud.githubusercontent.com/assets/16038202/12184279/3da2ba8e-b562-11e5-8ca6-623a250eb0d5.png">

I searched issues but couldn't find anything similar to this. 

There are the urls I am trying to load, by the way:

 let myURLs = ["http://imgur.com/a/VeCYT", "http://i.imgur.com/qYQDyFZ.jpg", "http://i.imgur.com/ebyIE6A.gif", "http://i.imgur.com/4Uv5XOK.jpg", "http://i.imgur.com/8j7tf4k.gif", "http://i.imgur.com/pAyQ2MA.gif", "http://i.imgur.com/rVNe8m3.jpg", "http://i.imgur.com/apL0vsp.jpg", "http://i.imgur.com/XsdBXMZ.png", "http://i.imgur.com/4j0slvch.gif", "http://imgur.com/a/KojbY", "http://i.imgur.com/3kSFolM.png", "http://imgur.com/a/LFQxo", "http://imgur.com/a/OMefU"]

**Comment by hirohisa** [comment]
Thank you for the issue, @JoeyBodnar .

I ask you that:
1. What version do you use?
2. Do you use `prepare` function for cells?

**Comment by JoeyBodnar** [comment]
I am using the latest version. I installed it into my project using cocoa pods just last week. And no, I have no used the prepare function for the cells. I simply called the imageView.load(http://www.myurl.com) method in cellForRowWithIndexPath.

**Comment by hirohisa** [comment]
Thanks. I investigate.

**Comment by hirohisa** [comment]
I use 0.8.7 to a project.
This issue did not occur in development.

Please try to implement to cells that:
```
    override func prepareForReuse() {
        imageView.image = nil
    }
```

**Comment by khovansky-al** [comment]
Encountered same issue, your solution helps.
Thanks a lot!

**Comment by hirohisa** [comment]
Thanks, @khovansky-al .

Could you show me code encounted the issue?

**Comment by khovansky-al** [comment]
@hirohisa basically it is a collection view with vertical screen-wide cells/tiles. Cell is covered with image that's being downloaded from server. Without that reuse fix cells start duplicating themselves if they disappear and then reappear back (for example you scroll down and then scroll up).

```
func collectionView(collectionView: UICollectionView, cellForItemAtIndexPath indexPath: NSIndexPath) -> UICollectionViewCell {
        let cell = collectionView.dequeueReusableCellWithReuseIdentifier("PlaceCell", forIndexPath: indexPath) as! PlacesCollectionViewCell
        
        cell.titleLabel.text = self.data[indexPath.row].name.uppercaseString
        
        cell.descriptionLabel.text = self.data[indexPath.row].shortDescription
        
        if (data[indexPath.row].placeImageURL != nil) && !data[indexPath.row].placeImageURL!.isEmpty {
            cell.placeImage.load(ServerManager.ServerURL + data[indexPath.row].placeImageURL!)
        }
        return cell
}
```

Also I found out that cache is not being busted if remote image has changed. But I guess this is a subject for another issue.

**Comment by hirohisa** [comment]
Maybe, I got it.
Bugfix to set image #76  .

#76 is fixed by version 0.8.9.
